modules/ROOT/pages/storage.adoc

```diff
@@ -0,0 +1,131 @@
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  disks:
+  - # The name of the primary block device. In virtio-based setups, this is
+    # likely `/dev/vda`. Elsewhere, it's likely `/dev/sda`.
+    device: /dev/vda
+    # We do not want to wipe the partition table since this is the primary
+    # device.
+    wipe_table: false
+    partitions:
+    - size_mib: 0
+      # Start at 5G so that we leave enough space for the root partition.
+      # See the important NOTE below about this.
+      start_mib: 5000
+      # We assign a descriptive label to the partition. This is important
+      # for referring to it in a device-agnostic way in other parts of the
+      # configuration.
+      label: var
+  filesystems:
+    - path: /var
+      device: /dev/disk/by-partlabel/var
+      # We can select the filesystem we'd like.
+      format: ext4
+      # Ask FCCT to generate a mount unit for us so that this filesystem gets
+      # mounted in the real root.
+      with_mount_unit: true
+----
+
+NOTE: The `start_mib` field is very important. In the future, we will make more clear how much space should be reserved for the root filesystem (see https://github.com/coreos/fedora-coreos-tracker/issues/586). For now, make sure to leave at least 5G.
+
+You can of course mount only a subset of `/var` into a separate partition. For example, to mount `/var/lib/containers`:
+
+.Adding a /var/lib/containers partition to the primary disk
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  disks:
+  - device: /dev/vda
+    wipe_table: false
+    partitions:
+    - size_mib: 0
+      start_mib: 5000
+      label: containers
+  filesystems:
+    - path: /var/lib/containers
+      device: /dev/disk/by-partlabel/containers
+      format: xfs
+      with_mount_unit: true
+----
+
+Alternatively, you can also mount storage from a separate disk. For example, here we mount `/var/log` from a partition on `/dev/vdb`:
+
+.Adding /var/log from a secondary disk
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  disks:
+  - device: /dev/vdb
+    wipe_table: false
+    partitions:
+    - size_mib: 0
+      start_mib: 0
+      label: log
+  filesystems:
+    - path: /var/log
+      device: /dev/disk/by-partlabel/log
+      format: xfs
+      with_mount_unit: true
+----
+
+== Reconfiguring the root filesystem
+
+It is possible to reconfigure the root filesystem itself. You can use the path `/dev/disk/by-label/root` to refer to the original root partition. You must ensure that the new filesystem also has a label of `/root`.
+
+NOTE: You must have at least 4G of RAM for root reprovisioning to work.
+
+Here's an example of moving from xfs to ext4, but reusing the same partition on the primary disk:
+
+.Changing the root filesystem to ext4
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  filesystems:
+    - device: /dev/disk/by-partlabel/root
+      wipe_filesystem: true
+      format: ext4
```

Comment:
I had to switch back to `ext4` here from `btrfs`. I think the latter would be a nicer example given the push in Fedora, though it needs https://github.com/coreos/fedora-coreos-config/pull/590. We can switch it once that merges.